src/ecmult_impl.h

```diff
@@ -544,11 +318,12 @@ static void secp256k1_ecmult_strauss_wnaf(const secp256k1_ecmult_context *ctx, c
             }
         }
         if (i < bits_ng_1 && (n = wnaf_ng_1[i])) {
-            ECMULT_TABLE_GET_GE_STORAGE(&tmpa, *ctx->pre_g, n, WINDOW_G);
+            ECMULT_TABLE_GET_GE_STORAGE(&tmpa, secp256k1_pre_g, n, WINDOW_G);
             secp256k1_gej_add_zinv_var(r, r, &tmpa, &Z);
         }
         if (i < bits_ng_128 && (n = wnaf_ng_128[i])) {
-            ECMULT_TABLE_GET_GE_STORAGE(&tmpa, *ctx->pre_g_128, n, WINDOW_G);
+            VERIFY_CHECK(secp256k1_pre_g_128 != NULL);
```

Comment:
What does this VERIFY_CHECK do? Isn't `secp256k1_pre_g_128` an array?

Reply:
This is outdated and needs to be removed.
I used to have `secp256k1_pre_g_128` equal to `NULL` for `EXHAUSTIVE_TEST_ORDER`, but later I rewrote it to define it even in that case, even though it is not used at the moment.

Reply:
Fixed.